### DISCUSSION TITLE
No file names in views?

### DISCUSSION THREAD

**Discussion by 0xDedSec42** [photoprism/photoprism#4567] [Q&A] [answered]
I installed Photoprism a view days ago and I really like it so far. But what really annoys me is that in the album view for example where I can see all my images with a lot of EXIF information like date, camera, lens, resolution and stuff, every image has the same title. It seems to be extracted from the filename. I usually do rename my images upon importing in to Lightroom to the session name. After the session name (?) there's the year of the capture. And it's the same for every single image which in my opinion doesn't make any sense. Why would every image have the same name?

![image](https://github.com/user-attachments/assets/e56c41cb-3708-406f-adc2-7b983a16b5e9)

I was searching Reddit for this and I'm not the first one with this question. Is there a way to show the filename instead?

**Comment by graciousgrey** [comment, marked as answer]
You are right, the card view displays the image title. If no title is present in the [photos metadata](https://www.photoprism.app/kb/metadata) it will be generated by PhotoPrism based on e.g. file and folder names, labels, locations or tagged people.

We do not show the filename per default because very often it is just a number.  If you change the sort order in search or the sort order of an album to "filename" you will additionally see the filename on the card. Config options to customize the information shown on the card view are planned for the future. 

- https://github.com/photoprism/photoprism/issues/1164

**Reply by 0xDedSec42** [reply]
Thank you for your answer. This feature would be highly appreciated. I personally do find the number much more relevant than having the same title on every single photo when looking at hundreds of photos.

**Reply by 0xDedSec42** [reply]
From a UX perspective, displaying the same combination of image name and capture year under each image feels redundant and does not help users distinguish between different images. It would be much more useful to show unique identifiers or attributes that aid in image identification. I strongly encourage reconsidering this approach and updating the display to something more informative and user-friendly.
